beluga_amcl/include/beluga_amcl/amcl_node_utils.hpp

```diff
@@ -46,7 +46,7 @@ inline Eigen::IOFormat make_eigen_comma_format() {
  * \return A vector of pairs of coordinates in the robot's reference frame.
  */
 std::vector<std::pair<double, double>> make_points_from_laser_scan(
-    const beluga_amcl::messages::LaserScan& laser_scan,
+    const beluga_ros::msg::LaserScan& laser_scan,
```

Comment:
@nahueespinosa meta: doesn't this function belong to `beluga_ros`?

Reply:
It does, I will address this in #170. Coming shortly.